.env.example

```diff
@@ -0,0 +1,4 @@
+dbHost="ezxssdb"
```

Comment:
This should remain in the final config file so that docker-compose users do not need to remove it.